content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,122 @@
+  [TURN server.](https://webrtc.org/getting-started/turn-server),
+  but other signaling channels, such as WebSocket running on a central server, can also be used.
+  Using a custom signaling protocol or a different signaling service is also possible.
+  Overall, this allows for faster and more efficient communication.
+
+- Security: WebRTC connections are encrypted using
+  [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security).
+
+Browsers expose an API to establish WebRTC connections. The
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection)
+API allows two applications on different endpoints to communicate.
+
+### Browser-to-Server
+
+libp2p WebRTC enables browsers to connect to public server nodes without those
+endpoints providing a TLS certificate within the browser's trustchain.
+
+{{< alert icon="" context="info" text="When connecting to a WebSocket server, browsers require the server to present a TLS certificate signed by a trusted CA (certificate authority). Few nodes have such a certificate, which is the reason that WebSocket never saw widespread adoption in the libp2p network. WebRTC and WebTransport support encrypted communication without requiring a signed certificate from a trusted CA." />}}
+
+In libp2p:
+
+- WebRTC multiaddresses are composed of a standard UDP multiaddr,
+  followed by `webrtc` and the `multihash` of the certificate that
+  the node uses, as such:
+  `/ip4/1.2.3.4/udp/1234/webrtc/certhash/<hash>/p2p/<peer-id>`;
+- WebRTC encrypts connections using DTLS. However, an additional handshake is required to
+  authenticate a peer's peer ID once the WebRTC connection has been established.
+
+A browser can connect to a server node without needing a trusted TLS
+certificate. View [this scenario](https://github.com/libp2p/specs/blob/master/webrtc/README.md#browser-to-public-server) on browser-to-server connection establishment
+in the technical specification.
+
+Contrary to the standard WebRTC handshake process, the browser and server do not
+exchange the SDP. Instead, they employ a technique known as
+[SDP munging](https://webrtc.github.io/samples/src/content/peerconnection/munge-sdp/).
+This technique allows the browser to simulate the exchange of an SDP, but in reality,
+it constructs it locally using the information provided by the server's multiaddress.
+
+{{< alert icon="" context="info" text="SDP (Session Description Protocol) is a protocol that is used to describe multimedia sessions. The SDP answer is a message sent by a server in response to an SDP offer from a client. The offer and answer are used to establish a session between a client and server, allowing them to exchange media." />}}
+
+When establishing a WebRTC connection, the browser and server perform a standard DTLS
+handshake as part of the connection setup. DTLS is similar to TLS but is designed to
+work on an unreliable transport instead of an ordered byte stream like TCP.
```

Comment:
"DTLS is similar to TLS but is designed to work on an unreliable transport instead of an ordered byte stream like TCP."
Should this explanation be on line 51, where we first introduce DTLS?

Reply:
Now we have it twice. I am fine with it. That said, I don't think line 51 needs to be repeated here again.